src/NuGet.Indexing/Properties/Strings.resx

```diff
@@ -141,6 +141,14 @@
     "universal windows platform"
   ],
 
+  "json.net" : [
```

Comment:
The `.` is the culprit - the `DotTokenizer` is first in the chain, which means there are never terms with a `.` coming through here.